### ISSUE TITLE
Sounds are missing

### ISSUE THREAD

**Issue by tsdgeos** [flathub/org.kde.kdiamond#6]
They play through sound notifications, there's several warnings about it on the command line

**Comment by Justinzobel** [comment]
I did some testing and added `--talk-name=org.freedesktop.Notifications` which allowed it to send notifications and the ogg files for the notifications are in the flatpak but they're not playing, I added `--socket=pulseaudio` as well but still no sounds. 🤔

**Comment by tsdgeos** [comment]
hmmm, the current flathub version works for me (make sure you raise the volume and are in a quiet room the sounds are very low in volume, but that's an existing issue out of flathub too)

**Comment by Justinzobel** [comment]
Yes they are very quiet. Even at full volume with my speakers. Closing issue.
